python/src/aiconfig/editor/server/queue_iterator.py

```diff
@@ -0,0 +1,41 @@
+"""Queue iterator for streaming. Can only process strings for now
+but in future will try to make it generic"""
+# import asyncio
+from queue import Queue
+# from typing import Generic, TypeVar
+
+# # TODO: Add generic typing for queue items
+# # (couldn't get sentinel value to work with generics)
+# T = TypeVar('T')
+STOP_STREAMING_SIGNAL = object() #sentinel value to indicate end of stream
```

Comment:
nit: object() is a weird sentinel value. Howa bout we make a Sentinel enum?

Reply:
Created issue to follow up in https://github.com/lastmile-ai/aiconfig/issues/794